docs/fastdds/dds_layer/topic/typeSupport/typeSupport.rst

```diff
@@ -1,4 +1,71 @@
-.. _dds_layer_topic_typeSupport:
+.. _dds_layer_definition_data_types:
+
+Definition of data types
+========================
+
+The definition of the data type exchanged in a :ref:`dds_layer_topic_topic` is divided in
+two classes: the :class:`TypeSupport` and the :class:`TopicDataType`.
+
+:class:`TopicDataType` describes the data type exchanged between a publication and a subscription, i.e.,
```

Comment:
The comma after "i.e." actually depends on whether you write in american english or british english. If it's american, you put it in, but if it's british, you don't. 
So they're both correct.

Reply:
always learning something